README.md

```diff
@@ -80,10 +80,12 @@ What the attributes mean:
 - `tags[].customImages[].dockerfileOptions[]`: The list of Dockerfile options, used to override base image
 - `patterns[]`: List of patterns. New tags matching one of these patterns will be automatically retagged.
 - `patterns[].pattern`: Valid semver condition to match tags.
+- `patterns[].filter`: Regex to filter tags before validating semver. If empty, filtering is not done. 
 - `patterns[].customImages[]`: Custom images as explained above.
 
 An image may define both `tags` and `patterns`.
```

Comment:
```suggestion
An image may define both `tags`, `patterns` and `filters`.
```

Reply:
tags and patterns are "top level" attributes of image. filter is inside patterns. I don't think it is correct add filters here.